Dom.zavd 1/Program.cs

```diff
@@ -0,0 +1,64 @@
+using ConsoleApp2;
+using System;
+using System.Security.Cryptography.X509Certificates;

```

Comment:
Remove this, you dont need this

Reply:
If I  remove "using ConsoleApp2" my program won`t work, because my "List" need class Car